frontend/tests/unit/components/HomePageButtons.spec.ts

```diff
@@ -0,0 +1,40 @@
+import Vue from "vue";
+import { createLocalVue, shallowMount } from "@vue/test-utils";
+import VueRouter from "vue-router";
+import HomePageButtons from "@/components/HomePageButtons.vue";
+import { router } from "@/router";
+
+describe("HomePageButtons", () => {
+  let localVue: typeof Vue;
+
+  beforeEach(() => {
+    localVue = createLocalVue();
+    localVue.use(VueRouter);
+  });
+
+  it("should render buttons that match TE claims", () => {
```

Comment:
These two test cases are identical except the role. Use Jest's [data-driven tests](https://jestjs.io/docs/en/api#2-testeach-table-name-fn-timeout).